cmd/jaeger/config-elasticsearch.yaml

```diff
@@ -17,6 +17,19 @@ extensions:
       some_storage:
         elasticsearch:
           index_prefix: "jaeger-main"
+#          indices:
+#            spans:
+#              date_layout: "2006-01-02"
+#              rollover_frequency: "hour"
```

Comment:
I don't think we should use comments, we can just keep the actual properties. But we need to use the same defaults as in v1 (I see that all rollovers default to daily).